src/frontend/src/optimizer/plan_node/stream_table_scan.rs

```diff
@@ -283,6 +283,13 @@ impl StreamTableScan {
     ) -> SchedulerResult<PbStreamNode> {
         use risingwave_pb::stream_plan::*;
 
+        let backfill_paused = self
```

Comment:
Currently we set the initial `backfill_paused` in frontend and then persist this value and never update it later. But as described in the other comment, after recovery if we don't reset this value to false, the paused backfill may never be unpaused.

So I am thinking about a cleaner implementation. Here in frontend, we may always initial the value to false, so that the persisted value is always false and after recovery all backfill nodes are not initially paused. The work of setting this `backfill_paused` field can be handled by barrier manager. After it resolve the initial `BackfillOrderState`, it set `backfill_paused=false` for nodes without dependency, and as `true` for nodes with dependency. We can update the node field via `visit_stream_node_mut`.

Reply:
Addressed in https://github.com/risingwavelabs/risingwave/pull/21474